composer.json

```diff
@@ -42,7 +42,7 @@
     }
   ],
   "require-dev": {
-    "doctrine/dbal": "~2.4",
+    "broadway/event-store-dbal": "^0.1",
```

Comment:
Should this be `~0.2`? I guess the event store lib might BC break in `0.x` releases?